package/gem2rpm.yml

```diff
@@ -0,0 +1,79 @@
+# ---
+# ## used by gem2rpm
+# :summary: this is a custom summary
+# ## used by gem2rpm
+# :description: |-
+#   this is a custom description
+#
+#   it can be multiline
+# ## used by gem2rpm
+# :license: MIT or Ruby
+# ## used by gem2rpm and gem_packages
+# :version_suffix: -x_y
+# ## used by gem2rpm and gem_packages
+:disable_docs: true
+# ## used by gem2rpm
+# :disable_automatic_rdoc_dep: true
+# ## used by gem2rpm
+:preamble: |-
+  # extra dependencies (not specified in gemspec)
+  BuildRequires: aspell-en
+  Requires:      aspell-en
+  # the rubocop shared config
+  BuildRequires: yast2-devtools
+  Requires:      yast2-devtools
+  # for the tests
+  BuildRequires: rubygem(yast-rake)
+  BuildRequires: rubygem(rubocop) = 0.29.1
+  BuildRequires: rubygem(gettext)
+  BuildRequires: rubygem(raspell)
+# ## used by gem2rpm
+# :patches:
+#   foo.patch: -p1
+#   bar.patch:
+# ## used by gem2rpm
+# :sources:
+# - foo.desktop
+# - bar.desktop
+# :gem_install_args: '....'
+# ## used by gem2rpm
+# :pre_install: |-
+#   %if 0%{?use_system_libev}
+#   export USE_VENDORED_LIBEV="no"
+#   %endif
+# ## used by gem2rpm
+# :post_install: |-
+#   # delete custom files here or do other fancy stuff
+#   install -D -m 0644 %{S:1} %{buildroot}%{_bindir}/gem2rpm-opensuse
+# ## used by gem2rpm
+:testsuite_command: |-
+   (pushd %{buildroot}%{gem_base}/gems/%{mod_full_name} && rake check:ci)
```

Comment:
Should be OK, because of the parenthesis the command should be run in a subshell.

JFYI: I reused the template from `gem2rpm`, I just simply replaced the `rake spec` command by `rake check:ci`, I suppose the template contains a reasonable example...